### PR TITLE
Leaves all investigations that the user is part of (Clears the left pane)

### PR DESCRIPTION
**Pull request by gshriki**
## Status
Ready

## Related Issues
N/A

## Description
Scripts that a user can run to leave all investigations he is a member of - to clean his left pane.

## Screenshots
N/A

## Related PRs
N/A

## Required version of Demisto
N/A

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Dependencies
- Demisto API is requires (using `dependson`)

## Additional changes
N/A

### CODE REVIEW

File: Scripts/script-DemistoLeaveAllInvestigations.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  var account = 'acc_' + incidents[0].account;
+
+  var res = executeCommand("getUsers", {current: true});
+  if (isError(res[0])) {
+      return { ContentsFormat: formats.text, Type: entryTypes.error, Contents: 'Cannot get current user.\nThis script cannot be executed within a playbook.' };
+  }
+
+  var username = res[0].Contents[0].username;
+
+  res = executeCommand("getIncidents", {query: 'investigation.users:"'+username+'" and status:Active and -owner:"'+username+'"', size: 10000});
```
Comment (Nitzperetz) [inline Scripts/script-DemistoLeaveAllInvestigations.yml]: Nice

File: Scripts/script-DemistoLeaveAllInvestigations.yml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  id: DemistoLeaveAllInvestigations
+  version: -1
+name: DemistoLeaveAllInvestigations
+script: |
+  function addEntry(text) {
+      entry =  [{ContentsFormat: formats.markdown, Type: entryTypes.note, Contents: text}];
+      executeCommand("addEntries", {"entries":JSON.stringify(entry)});
+  }
+
+  var account = 'acc_' + incidents[0].account;
```
Comment (glicht) [inline Scripts/script-DemistoLeaveAllInvestigations.yml]: acc_ is only for multi tenant. in non multi tenant this should be simply empty

```suggestion
  var account = incidents[0].account ?  'acc_' + incidents[0].account : "";
```

File: Scripts/script-DemistoLeaveAllInvestigations.yml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+              errors.push(res[0]);
+          }
+      });
+  }
+  return (errors.length === 0) ? 'Done' : errors;
+type: javascript
+tags:
+- DemistoAPI
+comment: Leaves all investigations that the user is part of (clears out the incidents
+  left pane). Incidnets that the user owner will remain in left pane.
```
Comment (glicht) [inline Scripts/script-DemistoLeaveAllInvestigations.yml]:
```suggestion
  the left pane). Incidents that the user owns will remain in the left pane. Requires Demisto REST API.
```

File: Scripts/script-DemistoLeaveAllInvestigations.yml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+enabled: true
+scripttarget: 0
+runonce: false
+runas: DBotWeakRole
+dependson:
+  must:
+  - demisto-api-post
+ReleaseNotes: Leaves all investigations that the user is part of (clears out the incidents
```
Comment (glicht) [inline Scripts/script-DemistoLeaveAllInvestigations.yml]:
```suggestion
ReleaseNotes: Leave all investigations that the user is part of
```

File: Scripts/script-DemistoLeaveAllInvestigations.yml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  left pane). Incidnets that the user owner will remain in left pane.
```
Comment (glicht) [inline Scripts/script-DemistoLeaveAllInvestigations.yml]:
```suggestion
 
```

**Comment by gshriki** [comment]
later

**Comment by glicht** [comment]
Reopenning and will commit chamges

**Pull request merged**